src/main/java/org/isf/medtype/service/MedicalTypeIoOperationRepository.java

```diff
@@ -25,11 +25,17 @@
 
 import org.isf.medtype.model.MedicalType;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Modifying;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
```

Comment:
This import is not used.

Reply:
Param is not used and removed, other are good